lib/ex_docker_build/docker_build.ex

```diff
@@ -123,8 +123,10 @@ defmodule ExDockerBuild.DockerBuild do
   # defp exec({"LABEL", args}, context, _path) do
   # end
 
-  # defp exec({"EXPOSE", args}, context, _path) do
-  # end
+  defp exec({"EXPOSE", args}, context, _path) do
+    Map.merge(context, %{"Ports" => args})
```

Comment:
can you add support for `"80/tcp"` and `"80"` with tcp as default? the same as Dockerfile specification https://docs.docker.com/engine/reference/builder/#expose so we can use

```dockerfile
EXPOSE 80
EXPOSE 80/tcp
EXPOSE 80/udp
```

Reply:
Like `defp exec({"EXPOSE", args} // {"EXPOSE", "80/tcp"}, context, _path)`

Reply:
do you mean like `defp exec({"EXPOSE", args} \\ {"EXPOSE", "80/tcp"}, context, _path) do`